components/director/pkg/jsonschema/validator.go

```diff
@@ -0,0 +1,35 @@
+package jsonschema
+
+import (
+	"github.com/xeipuuv/gojsonschema"
+)
+
+type validator struct {
+}
+
+func NewValidator() *validator {
+	return &validator{}
+}
+
+func (v *validator) Validate(jsonschema string, jsons ...string) (bool, error) {
+
+	stringLoader := gojsonschema.NewStringLoader(jsonschema)
+	schema, err := gojsonschema.NewSchema(stringLoader)
+	if err != nil {
+		return false, err
+	}
+
+	for _, json := range jsons {
```

Comment:
Maybe we should return information which json is not invalid?

Reply:
it is no longer a case since we decided to check only one json value here